react/features/participants-pane/functions.ts

```diff
@@ -257,7 +261,8 @@ export function getSortedParticipantIds(stateful: Object | Function): Array<stri
  * @param {string} searchString - The participants search string.
  * @returns {boolean}
  */
-export function participantMatchesSearch(participant: Object, searchString: string) {
+export function participantMatchesSearch(participant: { displayName: string; jid: string; name?: string; },
```

Comment:
Just to be sure, you did not use the Participant interface here, because you had jid instead of id?

Reply:
Yes and no. This function is used for Br Rooms participants as well as meeting participants. Br room participants only have displayName and jid so using the Participant interface wouldn't be very accurate